cl/disclosures/tasks.py

```diff
@@ -476,3 +523,27 @@ def import_disclosure(self, data: Dict[str, Union[str, int, list]]) -> None:
 
     # Save PDF content
     save_disclosure(extracted_data=content, disclosure=disclosure)
+    # Remove disclosure ID in redis for completed disclosure
+    interface.delete(disclosure_key)
+
+
+def has_been_extracted(data: Dict[str, Union[str, int, list]]) -> bool:
```

Comment:
I think I'd put this in `cl.disclosures.utils`. It's not a task and doing this is a bit weird:

`from cl.disclosures.tasks import has_been_extracted`

Reading that, I'd think that `has_been_extracted` would be a task, not a utility?

Reply:
that makes lots of sense - I had something gnawing about this

Reply:
I'll add has_been_pdfed as well